Makefile

```diff
@@ -1,4 +1,4 @@
-tag := $(patsubst v%,%,$(shell git describe --tags))
+tag := $(patsubst v%,%,$(shell git describe --tags --abbrev=0))
 version := $(tag:v%=%)
```

Comment:
Is this version variable still used?

Reply:
seems to be

Reply:
The tag is set to the git tag which in our case has the leading v. Version is the stripped version of the tag. It's used in a few places to set various versions